emulators/nonpareil/Portfile

```diff
@@ -75,6 +75,11 @@ post-patch {
     reinplace s|@PREFIX@|${prefix}|g ${worksrcpath}/src/SConscript
 }
 
+post-destroot {
+    # avoids conflict with nss
```

Comment:
Please add the URL to the ticket here in the comment.

Reply:
Updated